packages/components/src/menu/tests/chromatic/MenuTrigger.chroma.jsx

```diff
@@ -0,0 +1,296 @@
+import { Button, IconButton } from "@components/button";
+import { Item, Section } from "@components/collection";
+import { Menu, MenuTrigger } from "@components/menu";
+import { DisclosureArrow } from "@components/disclosure";
+import { Divider } from "@components/divider";
+import { HtmlButton } from "@components/html";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong